pkg/sdk/reconciler/reconciler.go

```diff
@@ -771,3 +775,11 @@ func (r *Reconciler) completeUpgrade(logger logr.Logger, cr client.Object, opera
 
 	return nil
 }
+
+func (r *Reconciler) setRecommendedLabels(cr client.Object, obj metav1.Object) {
+	labels := sdk.GetRecommendedLabelsFromCr(cr)
+
+	for k, v := range labels {
```

Comment:
What labels do we want to support here? I am not sure whether copying all the labels is the best approach. Why not to copy only what we need?

Reply:
`GetRecommendedLabelsFromCr()` will only gives us back specific labels (/part-of and /version) and not all:
https://github.com/kubevirt/controller-lifecycle-operator-sdk/pull/18/files#diff-e4dd60c5e2da412b62ac14f4d8c34b69f2688c3e7bd75b7a63aaf68898a9d13bR248

Reply:
OK